lib/fog/openstack/baremetal.rb

```diff
@@ -34,6 +34,8 @@ class OpenStack < Fog::Service
       request :list_nodes
       request :list_nodes_detailed
       request :patch_node
+      request :set_node_power_state
+      request :set_node_provision_state
```

Comment:
These should be set_power_state and set_provision_state and it would be best to clean up the comments:

      # request :set_power_state
      # request :provision_node